pyci/__init__.py

```diff
@@ -59,6 +61,7 @@
     "add_excitations",
     "add_seniorities",
     "add_gkci",
+    "add_energy",
```

Comment:
it's a good idea to call it cost function in general probably
I propose to change the name to add_cost